src/extension.ts

```diff
@@ -20,14 +20,17 @@ export function activate(context: vscode.ExtensionContext) {
 					if (lastSelectionTime != thisSelectionTime) return;
 
 					if (config.autoHideSideBar) {
-						//vscode.commands.executeCommand("workbench.files.action.focusFilesExplorer");
-						vscode.commands.executeCommand("workbench.extensions.action.showInstalledExtensions");
+                        //vscode.commands.executeCommand("workbench.files.action.focusFilesExplorer");
+						vscode.commands.executeCommand("workbench.view.search");
```

Comment:
Is there a reason you changed the toggle mechanism from the extensions panel to the search panel? I know I had problems with the explorer panel (caused not-linked-with-file tabs to close), but didn't know of any major problems with the extensions panel. (other than losing scroll of it)

Reply:
Oh, was it to solve the getViews error?